validation/src/main/scala/hmda/validation/dsl/PredicateRegEx.scala

```diff
@@ -4,8 +4,20 @@ import scala.language.implicitConversions
 import scala.util.matching.Regex
 
 object PredicateRegEx {
+  implicit def validCensusTractFormat: Predicate[String] = new Predicate[String] {
```

Comment:
This should get a test like the ones @nickgrippin wrote in #364 , but that doesn't need to block merging this PR. I created an issue to remind us: https://github.com/cfpb/hmda-platform/issues/367